.github/workflows/production-suite.yml

```diff
@@ -2,7 +2,21 @@ name: Zulip Production Suite
 
 on:
   push:
+    paths:
+      - "**/migrations/**"
+      - puppet/**
+      - scripts/**
+      - tools/**
   pull_request:
+    paths:
+      - "**/migrations/**"
+      - puppet/**
+      - scripts/**
+      - tools/**
+
+      # We want the production suite to run on PRs that
+      # update the CI config.
+      - .github/workflows/**
```

Comment:
This should be true for `push` as well, right?

Reply:
I can add it push as well sure. I though it would be just redundant so excluded it from pushes.